test/nodetool/test_version.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+from rest_api_mock import expected_request
+
+
+VERSION = "3.0.8"
```

Comment:
I just copied here whatever we currently print when running `nodetool version` on scylla. I can change it to be more obvious. I just wanted a constant to not have to (correctly) type the same version all over again.

Reply:
Yes, but:
1. you can move the constant inside the only function that uses it.
2. It doesn't need to be what we print on Scylla - the fact that it was confused me because I thought maybe you actually get this from Scylla so it should match. But it doesn't - you feed the fake response yourself :-)

Reply:
constant removed in v4